### PR TITLE
Added Contact Page

### PR DESCRIPTION
**Pull request by anurag0singh**
Closes: #56 

#### Describe the changes you've made

Added HTML, CSS, and Django backend code for the contact page.

## Checklist:

- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented on my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.

## Screenshots

image

### CODE REVIEW

**Comment by khannakshat7** [comment]
@anurag0singh Great frontend!!  Can u also add the mailing feature to mail to the admin?
Try to learn how to send mails in Django

**Comment by anurag0singh** [comment]
@khannakshat7 Yeah I am doing that only. Just wait for a few minutes.

**Comment by pranjalg13** [comment]
@anurag0singh Nice work, but just wanted to know the PR requests of #17 will be removed right? @khannakshat7 ?

**Comment by anurag0singh** [comment]
Screenshot (141)
Mail feature !

**Comment by anurag0singh** [comment]
> @anurag0singh Nice work, but just wanted to know the PR requests of #17 will be removed right? @khannakshat7?
> As I was working on the basis of that.

Sorry, @pranjalg13 if your hard work is wasted because of me. I apologize.

**Comment by pranjalg13** [comment]
> > @anurag0singh Nice work, but just wanted to know the PR requests of #17 will be removed right? @khannakshat7?
> > As I was working on the basis of that.
> 
> Sorry, @pranjalg13 if your hard work is wasted because of me. I apologize.

No worries, all good. 👍👍

**Comment by anurag0singh** [comment]
@khannakshat7 I have added the email feature. Please check it.

**Comment by nikhilkotiya** [comment]
@khannakshat7 Can i  contribute in this issue? I know how to send mails in Django

**Comment by anurag0singh** [comment]
> @khannakshat7 Can i contribute in this issue? I know how to send mails in Django

I have added the email feature and I am waiting for their review.
@nikhilkotiya

**Comment by shivaamm** [comment]
> > @khannakshat7 Can i contribute in this issue? I know how to send mails in Django
> 
> I have added the email feature and I am waiting for their review.
> @nikhilkotiya

Hey just want to confirm does this send emails directly to admin gmail? Cuz from above ss it looks like the mail is displayed on the cmd not the actual gmail.

**Comment by shivaamm** [comment]
I was also assigned to this backend contact form issue so if it's not fixed then i'll work on this

**Comment by anurag0singh** [comment]
@khannakshat7 Now I have added a Gmail server for mailing to admin.

**Comment by SUMEKAGARWAL** [comment]
@anurag0singh 
Hi Anurag, 
I am looking forward for your help in order to send a mail to the user.
Can I get your contact please?
Thank you!

File: Elecktra/settings.py
Comment (khannakshat7) [inline Elecktra/settings.py:82]: Pl. remove username from this line

Comment (anurag0singh) [reply Elecktra/settings.py]: @khannakshat7 Done.

**Review by shivaamm — APPROVED** [review]
I think with this PR the backend contact form issue will be solved.

**Comment by khannakshat7** [comment]
Error:
from electricity.forms import UserForm
ImportError: cannot import name 'UserForm'

@anurag0singh You didn't commit the forms.py file

**Comment by anurag0singh** [comment]
@khannakshat7 I have fixed the error please review it now.

**Comment by anurag0singh** [comment]
> Hey @anurag0singh We are having some urgent work to complete by Wednesday. Please cooperate and have patience. We will surely review and merge your PR.

The time is not the issue here. Actually accepting other PRs before mine is creating merge errors. I understand your condition and I apologize for the inconvenience caused due to me. @CodesbyUnnati

**Comment by CodesbyUnnati** [comment]
> > Hey @anurag0singh We are having some urgent work to complete by Wednesday. Please cooperate and have patience. We will surely review and merge your PR.
> 
> The time is not the issue here. Actually accepting other PRs before mine is creating merge errors. I understand your condition and I apologize for the inconvenience caused due to me. @CodesbyUnnati

Oh, okay I was not knowing that, we will merge the PRs after we will review your PR.

File: Elecktra/settings.py
Comment (khannakshat7) [reply Elecktra/settings.py:82]: This is still not removed

**Comment by khannakshat7** [comment]
@anurag0singh Great contribution dude!! 🙌
Don't forget to star⭐ the repo and follow us on GitHub

**Pull request merged**
